### PR TITLE
Add connect, ulx action, include reason in disconnect

### PR DESCRIPTION
**Pull request by sarahsturgeon**
WIP

### CODE REVIEW

File: web/discord_relay/queue.go [hunk 1]
```diff
@@ -88,17 +89,37 @@ func sendEvent(discord *discordgo.Session, event EventStruct, eventText string,
...
 func sendDisconnectMessage(discord *discordgo.Session, event EventStruct) {
-	sendEvent(discord, event, "Disconnected from the server", 0x990000, LEAVE_EMOJI)
+    reason := event.Data.Reason
+    steamLink := "https://steamid.io/lookup/" + event.Data.SteamId
+    message := "[Disconnected from the server](" + steamLink + ")"
+
+    if strings.Contains(reason, "\n") {
+        message = message + "\n```" + reason + "\n```"
+    } else {
+        message = message = " (" + reason + ")"
```
Comment (plally) [inline web/discord_relay/queue.go]:
```suggestion
        message = message + " (" + reason + ")"
```

**Comment by wrefgtzweve** [comment]
Would be great if the actual steamID could be showed in here, allows for easy searching and moderation

**Comment by sarahsturgeon** [comment]
That'll be included in this PR too

**Comment by wrefgtzweve** [comment]
Thoughts on splitting up the pr and adding ULX with another one? im willing to work on this but i feel like splitting it up might be easier

**Comment by sarahsturgeon** [comment]
I'll be reworking a big chunk of stuff in this one, makes the most sense to do it all at once.

I also have a good chunk of unpushed work here

**Comment by wrefgtzweve** [comment]
alr

**Pull request merged**